docs/ContentTypeParser.md

```diff
@@ -68,6 +72,40 @@ if (!fastify.hasContentTypeParser('application/jsoff')){
 }
 ```
 
+#### removeContentTypeParser
+
+With `removeContentTypeParser` a single or an array of content types can be removed. The method supports `string` and
+`RegExp` content types.
+
+```js
+fastify.addContentTypeParser('text/xml', function (request, payload, done) {
+  xmlParser(payload, function (err, body) {
+    done(err, body)
+  })
+})
+
+// Removes the both built-in content type parsers so that only the content type parser for text/html is available
+fastiy.removeContentTypeParser(['application/json', 'text/plain'])
+```
+
+#### resetContentTypeParsers
```

Comment:
I think the name can be improved. `Reset` sounds to me reset to some point which mostly reset to default value. 

It would be better if it called `removeAll`.

Reply:
Agreed!

Reply:
Makes sense to me! I will update the name